pkg/tsdb/elasticsearch/response_parser.go

```diff
@@ -157,6 +158,29 @@ func processLogsResponse(res *es.SearchResponse, target *Query, configuredFields
 	sortedPropNames := sortPropNames(propNames, configuredFields, true)
 	fields := processDocsToDataFrameFields(docs, sortedPropNames, configuredFields)
 
+	// copy `_id` field to `id` field
```

Comment:
Instead of doing another loop in response processing, what if we re-used the loop where we are turning `hits` into `docs` instead. We are already preparing there fields such as `level` and others. 

```
		_, ok := doc["id"] 
		if !ok {
			_, okId := doc["_id"]
			_, okIndex := doc["_index"]
			if okId && okIndex {
				doc["id"] = fmt.Sprintf("%v", doc["_id"]) + "#" + fmt.Sprintf("%v", doc["_index"])
			}
		}
```

Reply:
In my previous iteration I had something like that, but that would mean we are adding `id` to all responses, i.e. also `raw document` response and I did not really like that. WDYT?

Reply:
We could add a flag to `processDocsToDataFrameFields` though if `id` should be added or not.

Reply:
> but that would mean we are adding id to all responses, i.e. also raw document...

I don't think that would be a case, because we would be doing this in `processLogsResponse`. `processRawDataResponse` has its own logic to add fields, as there are some differences - renaming of level field, creating searchWords,...

Reply:
Oh, I see. Just couple of lines above my changes. Good call, will do later!

Reply:
Thank you Ivana, that was a great suggestion!

Added in cfcc512820a91a12b3f0be8825d10d1fe8389f68